OpenRA.Mods.Common/Widgets/WidgetUtils.cs

```diff
@@ -324,19 +333,26 @@ public static void BindPlayerNameAndStatus(LabelWidget label, Player p)
 		{
 			var client = p.World.LobbyInfo.ClientWithIndex(p.ClientIndex);
 			var nameFont = Game.Renderer.Fonts[label.Font];
-			var name = new CachedTransform<(string Name, WinState WinState, Session.ClientState ClientState), string>(c =>
+			var name = new CachedTransform<(WinState WinState, Session.ClientState ClientState), string>(c =>
 			{
-				var suffix = c.WinState == WinState.Undefined ? "" : " (" + c.WinState + ")";
-				if (c.ClientState == Session.ClientState.Disconnected)
-					suffix = " (Gone)";
+				var text = p.ResolvedPlayerName;
+
+				if (c.WinState == WinState.Won)
+					text += $" ({TranslationProvider.GetString(Won)})";
+				else if (c.WinState == WinState.Lost)
+					text += $" ({TranslationProvider.GetString(Lost)})";
+
+				if (client.State == Session.ClientState.Disconnected)
+					text += $" ({TranslationProvider.GetString(Gone)})";
```

Comment:
the original behaviour only uses 1 suffix, and this implementation now allows multiple. If we keep the original behaviour then we can keep formatting on the tfl side